src/main/java/seedu/taassist/logic/commands/AddCommand.java

```diff
@@ -36,32 +36,32 @@ public class AddCommand extends Command {
     public static final String MESSAGE_SUCCESS = "New student added: %1$s";
     public static final String MESSAGE_DUPLICATE_STUDENT = "This student already exists in TA-Assist";
 
-    private final Student toAdd;
+    private final Student student;
```

Comment:
We might want to standardize the variable names afterwards with the group to decide how we should name the variables so that our code will look as if 1 person wrote them. But thanks for pointing that out!